listenbrainz/webserver/views/user.py

```diff
@@ -46,7 +47,7 @@ def lastfmscraper(user_name):
         import_url="{}/1/latest-import".format(config.BETA_URL),
         user_token=user_token,
         lastfm_username=lastfm_username,
-        user_name=user_name,
+        user_name=escape(user_name), # need to escape user_name here because other wise jinja doesn't handle usernames with backslashes correctly
```

Comment:
Can we have a few tests with this template to make sure it renders various usernames correctly?